guides/source/security.md

```diff
@@ -949,6 +949,27 @@ Location: http://www.malicious.tld
 
 So _attack vectors for Header Injection are based on the injection of CRLF characters in a header field._ And what could an attacker do with a false redirection? They could redirect to a phishing site that looks the same as yours, but ask to login again (and sends the login credentials to the attacker). Or they could install malicious software through browser security holes on that site. Rails 2.1.2 escapes these characters for the Location field in the `redirect_to` method. _Make sure you do it yourself when you build other header fields with user input._
 
+#### DNS rebinding and Host header attacks
```

Comment:
Most words are capitalized in headings:
https://guides.rubyonrails.org/ruby_on_rails_guides_guidelines.html#headings
```suggestion
#### DNS Rebinding and Host Header Attacks
```